test/clustertest/tests/BadCommandTest.cpp

```diff
@@ -30,7 +30,7 @@ struct BadCommandTest : tpunit::TestFixture {
         try {
             leader.executeWaitVerifyContent(cmd, "500 Unhandled Exception");
         } catch (...) {
-            cout << "failing in first block." << endl;
+            cout << "[BadCommandTest] failing in first block." << endl;
```

Comment:
since you added this to every file it really makes more sense to make it a function or generalize it. I actually think you could remove the `cout` and replace it with logs, I'm pretty sure we fixed that.

Reply:
Is this a blocker? I'm not inclined to make a global error handler for our test harness.